site/en/guides/thread-primer/node-roles-and-types.md

```diff
@@ -151,9 +151,201 @@ What you learned:
+    What are they?</div>
+    <div>
+      <div>Child Node.</div>
+      <div>Incorrect.</div>
+    </div>
+    <div correct>
+      <div>Router.</div>
+      <div>Correct.</div>
+    </div>
+    <div correct>
+      <div>End Device.</div>
+      <div>Correct.</div>
+    </div>
+    <div>
+      <div>Gateway.</div>
+      <div>Incorrect.</div>
+    </div>
+  </devsite-multiple-choice>
+</div>
+
+<div>
+  <devsite-multiple-choice>
+    <div>What are the two primary types of Thread device?</div>
+    <div correct>
+      <div>Minimal Thread Device (MTD).</div>
+      <div>Correct.</div>
+    </div>
+    <div correct>
+      <div>Full Thread Device (FTD).</div>
+      <div>Correct.</div>
+    </div>
+    <div>
+      <div>Miniscule Thread Device (MTD).</div>
+      <div>Incorrect.</div>
+    </div>
+    <div>
+      <div>Sleepy End Device (SED).</div>
+      <div>Incorrect.</div>
+    </div>
+  </devsite-multiple-choice>
+</div>
+
+
+<div>
+  <devsite-multiple-choice>
+    <div>Which of the following statements about Routers is not true?</div>
+    <div correct>
+      <div>A Router can disable its transceiver to reduce power.</div>
+      <div>Devices that are functioning as Routers do not disable their
+        transceiver. (If they did, they'd be unable to function properly as a
+        Router.)</div>
+    </div>
+    <div>
+      <div>A Router forwards packets for network devices.</div>
+      <div>This statement is true.</div>
+    </div>
+    <div>
+      <div>A Router keeps its transceiver enabled at all times.</div>
+      <div>This statement is true. In order to function properly as a Router,
+      a device must keep its transceiver online at all times.</div>
+    </div>
+    <div>
+      <div>A Router provides secure commissioning services for devices trying 
+      to join the network.</div>
+      <div>This statement is true. Commissioning is an important function of a
+      Thread Router.</div>
+    </div>
+  </devsite-multiple-choice>
+</div>
+
+<div>
+  <devsite-multiple-choice>
+    <div>When can a device upgrade itself to a Router?</div>
+    <div correct>
+      <div>When it is a REED and it is the only node in reach of a new End
+        Device seeking to join the Thread network.</div>
+      <div>That's right. Under these circumstances, a REED can promote itself
+        to a Router.</div>
+    </div>
+    <div>
+      <div>When it is an End Device seeking to join the Thread network.</div>
+      <div>Incorrect.</div>
+    </div>
+    <div>
+      <div>When it is a REED and the Thread network has merged with a larger
+        network.</div>
+      <div>Incorrect.</div>
+    </div>
+  </devsite-multiple-choice>
+</div>
+
+<div>
+  <devsite-multiple-choice>
+    <div>When can a Router cause itself to stop acting as a Router?</div>
+    <div correct>
+      <div>When it has no children.</div>
+      <div>That's correct. A Router with no children is allowed to revert to
```

Comment:
```suggestion
      <div>That's correct. A Router with no children may revert to
```
There are other conditions that need to be satisfied (e.g. it does not cause a partition in the Thread network).